src/hooks/screen/useChatInfoHook.ts

```diff
@@ -100,6 +98,7 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   }, []);
 
   return {
+    isLoadingFetchingChannelDetail,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The inclusion of `isLoadingFetchingChannelDetail` in the return statement allows other components or hooks that use `useChatInfoScreenHook()` to access this state variable and adjust their behavior accordingly. This is a good change as it increases the flexibility and usability of this hook.

```diff
   return {
+    isLoadingFetchingChannelDetail,
     channelInfo,
     goBack,
     onContactPressed,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->